src/schema/decimal.rs

```diff
@@ -0,0 +1,84 @@
+use crate::utils;
+use bigdecimal::BigDecimal;
+use num_bigint::ToBigInt;
```

Comment:
Need to move this import under `mod tests`
```
warning: unused import: `num_bigint::ToBigInt`
```

Reply:
Done